### PR TITLE
Cherry-pick of commits from old PR #13

### PR DESCRIPTION
**Pull request by esben**
This is a cherry pick of most of the commits in the old PR #13 from @diorcety.

### CODE REVIEW

File: recipes/binutils/binutils.inc [hunk 1]
```diff
@@ -12,12 +12,12 @@ BUGTRACKER = "http://sourceware.org/bugzilla/"
...
 RECIPE_TYPES = "cross sdk-cross machine canadian-cross"
 
-inherit autotools make-vpath gettext
+inherit autotools make-vpath
 
 require conf/fetch/gnu.conf
 SRC_URI = "${GNU_MIRROR}/binutils/binutils-${PV}.tar.bz2 "
 
-DEPENDS = "native:flex native:bison"
+DEPENDS = "native:flex native:bison native:makeinfo native:m4"
```
Comment (esben) [inline recipes/binutils/binutils.inc]: @diorcety where do you expect to get native:makeinfo from?  Did you send a PR with makeinfo recipe, or should it be added to ASSUME_PROVIDED?

Comment (diorcety) [inline recipes/binutils/binutils.inc]: https://github.com/diorcety/oe-lite-base/commit/b2f8aefabd3f3dfa29b5c876afc9b59f15770fcc from https://github.com/oe-lite/base/pull/22

Comment (esben) [inline recipes/binutils/binutils.inc]: Ok.  We should get that merged then :)

But what about adding native:makeinfo to conf/provided/minimal.conf also?

Comment (diorcety) [inline recipes/binutils/binutils.inc]: Can be. But all these PRs have goal to have the minimal provided tool (a minimal minimal)

Comment (esben) [inline recipes/binutils/binutils.inc]: Yes.  Maybe we should only add it to default.conf and all.conf instead then.
That is probably both the right thing, and will allow us to build test this PR without the texinfo recipe merged in OE-lite/base.

Comment (diorcety) [inline recipes/binutils/binutils.inc]: I agreed. So we let the PR like this and i create the PR for texinfo into base?

Comment (esben) [inline recipes/binutils/binutils.inc]: I think you should add native:makeinfo to default.conf and all.conf in this PR, so we can merge it independently of changes in base.

And creating a PR for texinfo in base would be great.

**Pull request merged**